### PR TITLE
Restore missing app.cfg.example settings.  Make changes from Joe revi…

### PR DESCRIPTION
**Pull request by kburke**
This branch addresses changes Joe requested on https://github.com/sennetconsortium/ingest-api/pull/6, which I will cancel because we did the switch to Blueprint yesterday.  It also has the missing app.cfg variables for the file system operations.

### CODE REVIEW

**Review by SamSedivy — APPROVED** [review]

File: src/instance/app.cfg.example [hunk 1]
```diff
@@ -2,6 +2,25 @@
 APP_CLIENT_ID = ''
 APP_CLIENT_SECRET = ''
 
+# File system settings (committed settings below are for DEV)
+GLOBUS_ENDPOINT_FILEPATH = '/hive/sennet-dev/data/protected'
```
Comment (yuanzhou) [inline src/instance/app.cfg.example:6]: @kburke per Bill's answer, we should use `/codcc/dev-test/data-dev` as the base directory for these file system settings.

File: src/instance/app.cfg.example [hunk 2]
```diff
@@ -2,6 +2,25 @@
...
+SENNET_WEBSERVICE_FILEPATH = '/hive/sennet-dev/assets'
+WORKFLOW_SCRATCH = '/hive/sennet-dev/scratch'
```
Comment (yuanzhou) [inline src/instance/app.cfg.example:8]: SenNet doesn't use Airflow, thus no need for this scratch dir config item.

**Review by yuanzhou — CHANGES_REQUESTED** [review]

File: src/instance/app.cfg.example (hunk 2 above)
Comment (kburke) [reply src/instance/app.cfg.example]: Change pushed.

File: src/instance/app.cfg.example (hunk 1 above)
Comment (kburke) [reply src/instance/app.cfg.example]: Change pushed.

**Review by yuanzhou — APPROVED** [review]

**Pull request merged**